polyfill/test/ZonedDateTime/prototype/toPlainDateTime/plain-custom-timezone.js

```diff
@@ -8,15 +8,14 @@ includes: [compareArray.js]
 
 const actual = [];
 const expected = [
-  "get timeZone.getPlainDateTimeFor",
-  "call timeZone.getPlainDateTimeFor",
+  "get timeZone.getOffsetNanosecondsFor",
+  "call timeZone.getOffsetNanosecondsFor",
 ];
 
-const dateTime = Temporal.PlainDateTime.from("1963-07-02T12:00:00.987654321");
 const timeZone = new Proxy({
-  getPlainDateTimeFor() {
-    actual.push("call timeZone.getPlainDateTimeFor");
-    return dateTime;
+  getOffsetNanosecondsFor() {
+    actual.push("call timeZone.getOffsetNanosecondsFor");
+    return -8735135802468;
```

Comment:
are these numbers just different keyboard mashings?

Reply:
I don't know where `160583136123456789n` came from below, but this is the number that yields a PlainDateTime whose time is 12:00:00.987654321.